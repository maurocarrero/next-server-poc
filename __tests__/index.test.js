import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../pages/index';

it('should match', () => {
  const { container } = render(<HomePage />);
  expect(container).toMatchSnapshot();
});
