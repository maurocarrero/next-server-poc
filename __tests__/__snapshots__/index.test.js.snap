// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match 1`] = `
<div>
  <section
    class="container"
  >
    <header>
      <h1>
        Movies
      </h1>
    </header>
    <section
      class="row"
    />
  </section>
</div>
`;
